### PR TITLE
Correct and lighten text, move troubleshooting up

### PR DESCRIPTION
**Pull request by grischard**

### CODE REVIEW

**Comment by grischard** [comment]
Same as on main branch, sorry :/

File: src/index.html [hunk 1]
```diff
@@ -110,90 +110,88 @@
...
                     </div>
                 </div>
                 <div class="item">
                     <i class="info icon"></i>
                     <div class="content">
-                        <div class="header">How to save the certificate on Android?</div>
+                        <div class="header">Can I save the certificate on Android?</div>
                         <div class="description">
-                            <p>On Android, you can save the .pkpass file and open it with an application that supports this format. Here are some:</p>
+                            <p>You can save the .pkpass file and open it with an application that supports this format, for example:</p>
```
Comment (clawfire) [inline src/index.html:173]: Autant y aller fort et dire "Yes you can ! Save the ..." 😁  #yeswecan

File: src/index.html [hunk 2]
```diff
@@ -110,90 +110,88 @@
...
+                        <div class="description">If you like the app, don't hesitate to share it with your friends and family. Then, if you want to help more, <a href="https://github.com/clawfire/covid19-passbook-generator/issues?q=is%3Aopen" title="View all the opened issues">take a look at the issues</a>. It's not just about tech; there's also copywriting tasks, design, etc. You can start <a href="https://github.com/clawfire/covid19-passbook-generator/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22" title="See the issues recommended to start with">with those issues</a> if you want to get familiar with the project.
```
Comment (clawfire) [inline src/index.html:194]: ooooh nice shot

**Review by clawfire — COMMENTED** [review]
Fine by me, what @biou think about the FAQ new order?

**Comment by clawfire** [comment]
P.S: @biou just tell me if it's ok, I'll do the tricky rebase conflict if you want :p

File: src/index.html (hunk 1 above)
Comment (biou) [reply src/index.html:173]: :-)

**Comment by biou** [comment]
That's ok for me! This is probably the most frequently asked question :)

**Review by clawfire — APPROVED** [review]

**Pull request merged**